MinVer.Lib/MinVer.Lib.csproj

```diff
@@ -6,9 +6,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="LibGit2Sharp" Version="0.25.3" />
+    <PackageReference Include="LibGit2Sharp" Version="0.26.0-preview-0070" />

```

Comment:
@bording what's your view on MinVer taking this pre-release dep?

Reply:
I think it should be fine. Preview 70 is also the one that removes the curl dependency, so that is another reason to do it.

Reply:
It would fix this issue: https://github.com/adamralph/minver/issues/171

Reply:
Thanks @bording.

Thanks @slang25, that's true. I've raised #179 to update the README.